docs/env_vars.rst

```diff
@@ -163,9 +163,9 @@ Values: 0, 1
 
 Default: 0
 
-If set, DALI will use the dynamic executor (as if ``exec_dynamic=True`` was set in the  Pipeline)
+If set, DALI will use the default execution model (as if ``exec_dynamic=True`` was set in the Pipeline)
```

Comment:
@mzient will this flag overwrite the `exec_dynamic=False`?
I would also add a note that `0` lets `exec_dynamic` to decide.

Reply:
Yes, it will override `exec_dymamic=False` if the remaining flags are compatible.